Exercises/variables-declaration/variables-declaration.js

```diff
@@ -0,0 +1,37 @@
+//1. Declare a variable which will be encapsulated from global scope
+
+let price = 50; 
+
+function TotalCount () {
+  let discount = 17; 
```

Comment:
use encapsulation from global scope without functions but using `let/const` and theirs features